integration-tests/contract-permits.spec.ts

```diff
@@ -36,6 +36,8 @@ CONFIGS().forEach(({ lib, rpc, setup, createAddress }) => {
       expect(op.hash).toBeDefined();
       expect(op.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY);
       const permit_contract = await op.contract();
+      
+      console.log("permit contract address:"+permit_contract.address)
```

Comment:
I think you forgot to remove the console.log here

Reply:
fixed